### DISCUSSION TITLE
Procurement documents in subtype T01

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#989] [Q&A] [answered]
Is there a reason why it is forbidden to provide information on procurement documents in subtype T01? Wouldn't it make sense to include the GR-Part-ProcurementDocuments group here in the same way as in subtype 4,5 and 6?

**Comment by YvesJo** [comment, marked as answer]
For this non-SDK related question, please get in touch with the helpdesk who will track your request. Discussions internal to the European Commission have to take place before being able to add the fields.
